app/core/config.py

```diff
@@ -6,14 +6,16 @@
 if not SQLALCHEMY_DATABASE_URI:
     raise RuntimeError("'{DATABASE_URL}' environment variable must be set")
 
-PUBLIC_APP_URL = os.environ["PUBLIC_APP_URL"].rstrip("/")
+PUBLIC_APP_URL = os.getenv("PUBLIC_APP_URL", "")
+if PUBLIC_APP_URL:
+    PUBLIC_APP_URL = PUBLIC_APP_URL.rstrip("/")
 API_V1_STR = "/api/v1"
 
 # OpenSearch Config
-OPENSEARCH_URL = os.environ["OPENSEARCH_URL"]
-OPENSEARCH_USERNAME = os.environ["OPENSEARCH_USER"]
-OPENSEARCH_PASSWORD = os.environ["OPENSEARCH_PASSWORD"]
-OPENSEARCH_INDEX_PREFIX = os.environ["OPENSEARCH_INDEX_PREFIX"]
+OPENSEARCH_URL = os.getenv("OPENSEARCH_URL", "")
```

Comment:
Why this change, when would we want empty values for these as default?

Reply:
Apologies, this config file snuck into the PR. Will remove.